contracts/utility-token/UtilityTokenInterface.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @title Utility token interface - It is an interface providing methods to
+ *        burn the tokens.
+ */
+contract UtilityTokenInterface {
```

Comment:
I don't think that `UtilityTokenInterface` is needed. 
`burn` and `burnFrom` are already implemented in ERC20Token, these functions must be part of `ERC20I`

Reply:
Added `burn` and `burnFrom` in ERC20I and removed `UtilityTokenInterface`.